src/server.c

```diff
@@ -2604,6 +2604,20 @@ void setImplictACLCategories(struct redisCommand *c) {
         c->acl_categories |= ACL_CATEGORY_SLOW;
 }
 
+/* Recursively populate the args structure and return the number of args. */
+int populateArgsStructure(struct redisCommandArg *args) {
```

Comment:
seems like the only thing this function does is populate num_args, no? the name is misleading

Reply:
i was thinking that maybe it will do other things in the future